src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -769,6 +765,14 @@ def computer_export_setup(computer, output_file, sort):
         'prepend_text': computer.get_prepend_text(),
         'append_text': computer.get_append_text(),
     }
+
+    try:
+        output_file = generate_validate_output_file(
+            output_file=output_file, entity_label=computer.label, overwrite=overwrite, appendix='-setup'
+        )
+    except (FileExistsError, IsADirectoryError) as e:
+        echo.echo_critical(message=e)
+
```

Comment:
Maybe I'm wrong, but `generate_validate_output_file` seems like a redundant function :) ?
It raises two **built-in** errors `IsADirectoryError` & `FileExistsError` that are being handled all together right here.
I mean `write_text` itself raises `FileExistsError` , and in my opinion `IsADirectoryError` doesn't need to be raised at all. 

Apart from that, also has `output_file` both as input and output. Feels a bit weird :)

Reply:
The other functionality of the function is to define the default value of the output file in case one hasn't been specified by the user. This is also why the function returns the output file